### ISSUE TITLE
`Site name` cannot be hidden in Innovation theme regardless of appearance setting

### ISSUE THREAD

**Issue by ryanjbonnell** [ASU/webspark-drops-drupal7#58]
**tl;dr**

You can refer to [pull request #56][1] for more information on this issue along with a working solution.

--

This issue occurs "out of the box" using all default settings and is easily reproducible:

1. Spin-up or use an existing Webspark distribution
2. Set the default theme to _Innovation 7.x-1.x_
3. Navigate to Admin > Appearance > Settings at `/admin/appearance/settings/innovation`
4. Toggle the "Site name" checkbox to off (empty)
5. Save any/all changes
6. Clear Drupal's theme registry and site cache at `/admin/config/development/performance`
7. View the front-facing, public site 

Notice that regardless of the appearance setting for "site name" (true/false or checked/unchecked), the theme displays the Drupal site name in the header region.

![screenshot][2]

This behavior is due to the following logic bug, which outputs a duplicate 'class' attribute on the H1 element in [`page.tpl.php`][2] at line 90:

```php
<h1 class="header__sitename"<?php if ($hide_site_name) { print ' class="element-invisible"'; } ?>>
```

_When `$hide_site_name` evaluates to true, the &lt;h1&gt; tag gets duplicate class attributes, causing the second attribute to be ignored by user agents and never allowing the CSS ‘hook’ class of `element-invisible` to be used._

Tested using ASU Webspark 1.13.2 "Oklahoma" and ASU Innovation theme 7.x-1.0-beta6.

[1]: https://github.com/ASU/webspark-drops-drupal7/pull/56
[2]: http://i.stack.imgur.com/Mg9n6.png
[3]: https://github.com/ryanjbonnell/webspark-drops-drupal7/commit/f52430412438ef132050cdd3ef8c47cddbeeed83?diff=unified

**Comment by kjwaddel** [comment]
Refer to pull request #56.
